sql/updates/world/3.3.5/2023_xx_07_xx_world.sql

```diff
@@ -0,0 +1,15 @@
+UPDATE `creature_template` SET `ScriptName`='npc_marmot' WHERE `entry`=22189;
+
+DELETE FROM `spell_script_names` WHERE `ScriptName`='spell_q10720_the_smallest_creature';
+DELETE FROM `spell_script_names` WHERE `ScriptName`='spell_coax_marmot';
+INSERT INTO `spell_script_names` (`spell_id`,`ScriptName`) VALUES
+(38629, 'spell_q10720_the_smallest_creature'),
+(38544, 'spell_coax_marmot');
+
+DELETE FROM `conditions` WHERE (`SourceTypeOrReferenceId` = 13) AND (`SourceGroup` = 1) AND (`SourceEntry` = 38629) AND (`SourceId` = 0) AND (`ElseGroup` = 0) AND (`ConditionTypeOrReference` = 31) AND (`ConditionTarget` = 0) AND (`ConditionValue1` = 3) AND (`ConditionValue2` = 22356) AND (`ConditionValue3` = 0);
+DELETE FROM `conditions` WHERE (`SourceTypeOrReferenceId` = 13) AND (`SourceGroup` = 1) AND (`SourceEntry` = 38629) AND (`SourceId` = 0) AND (`ElseGroup` = 1) AND (`ConditionTypeOrReference` = 31) AND (`ConditionTarget` = 0) AND (`ConditionValue1` = 3) AND (`ConditionValue2` = 22367) AND (`ConditionValue3` = 0);
+DELETE FROM `conditions` WHERE (`SourceTypeOrReferenceId` = 13) AND (`SourceGroup` = 1) AND (`SourceEntry` = 38629) AND (`SourceId` = 0) AND (`ElseGroup` = 2) AND (`ConditionTypeOrReference` = 31) AND (`ConditionTarget` = 0) AND (`ConditionValue1` = 3) AND (`ConditionValue2` = 22368) AND (`ConditionValue3` = 0);
```

Comment:
Delete by SourceTypeOrReferenceId, SourceGroup and SourceEntry only (SourceId is always 0 for this SourceTypeOrReferenceId)